doc/api/index.rst

```diff
@@ -70,6 +70,14 @@ Input and Output
 
     load_icgem_gdf
 
+Synthetic models and surveys
+----------------------------
+.. autosummary::
+   :toctree: generated/
+
+    synthetic.airborne_survey
+    synthetic.ground_survey
+
```

Comment:
To be clear, I'm only talking about the submodule. The API section could be the same.

Reply:
I created the new `synthetic` submodule with the intention to store both synthetic surveys and synthetic models.
Because these are not independent datasets I held myself of adding these two functions under `datasets`. 
What do you think?

Reply:
I would prefer keeping `datasets` only for real world *data*, such as the ones stored on Rockhound.
And store anything related on synthetic modelling on the new `synthetic` submodule.
We will see how this works in the future, and until `1.0.0` we can change it if we don't like it.

Reply:
I bow to your wisdom :bowing_man: